changelog/unreleased/enhancement-move-file-drag-drop

```diff
@@ -0,0 +1,6 @@
+Enhancement: Move file via drag and drop
+
+We've added drag and drop file moving for the file view.
```

Comment:
```suggestion
We've added moving files and folders via drag and drop to the files table view.
```

Reply:
Not sure if we support files _and_ folders already though?

Reply:
yeah you are right, yes we support both, moving files and folders